components/admin_console/reset_email_modal/reset_email_modal.jsx

```diff
@@ -121,7 +121,6 @@ export default class ResetEmailModal extends React.Component {
                                         type='email'
                                         ref='email'
                                         className='form-control'
-                                        maxLength='22'
```

Comment:
Should we copy the paradigm in [signup_email.jsx](https://github.com/mattermost/mattermost-webapp/blob/master/components/signup/signup_email/signup_email.jsx#L351) and just set this to `128`?

Reply:
Will set it to `128` then.

Reply:
Cool, thanks!
(Don't know why did I mention admin console link above.  I thought it's different from this one.)